src/parameterized_tendencies/radiation/radiation.jl

```diff
@@ -141,15 +117,17 @@ end
 
 function rrtmgp_model_kwargs(
     space,
+    params,
+    time_varying_trace_gases,
     radiation_mode::RRTMGPI.AbstractRRTMGPMode,
-    ozone::AbstractOzone,
-    co2::AbstractCO2,
     include_z::Bool,
 )
     ᶜspace = Spaces.center_space(space)
     ᶠspace = Spaces.face_space(space)
     FT = Spaces.undertype(space)
 
+    trace_gas_params = CAP.trace_gas_params(params)
```

Comment:
could also just pass `trace_gas_params` into `rrtmgp_model_kwargs` directly, since `params` isn't used for anything else in this function.